### PR TITLE
Spring 23 Updates

### LINKED ISSUE #15: Missing Dockerfile
- [ ] A dockerfile could be added for easy deployment puposes.

### PR DESCRIPTION
**Pull request by rajivharlalka**
- [x] Update course lists
- [x] Add Dockerfile
- [x] Update minor list

Also, Happy new year 🎉 

Closes #15

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -12,4 +12,4 @@ bs4 = "*"
 pylint = "*"
 
 [requires]
-python_version = "3.6"
```
Comment (thealphadollar) [inline Pipfile:15]: Keeping a fixed Python version is better - whichever latest version works correctly.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+RUN PIPENV_VENV_IN_PROJECT=1 pipenv install --deploy
+
+FROM base AS runtime
+COPY --from=python-deps /.venv /.venv
+ENV PATH="/.venv/bin:$PATH"
+COPY . .
+
+EXPOSE 5000
+
+ENTRYPOINT ["python3", "app.py"]
```
Comment (thealphadollar) [inline Dockerfile]: nitpick - let's add a new line here.

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Thanks for the PR, Rajiv. I've left some comments - also, I think we can remove the .vscode/settings.json from committed files.

File: Dockerfile (hunk 2 above)
Comment (rajivharlalka) [reply Dockerfile]: Just for knowledge , do we have a new line at the end of file because some interpreters consider the line last new line as the EOF?

**Comment by rajivharlalka** [comment]
Hi Shivam, Have already removed the .vscode/settings file in the PR.

File: Dockerfile (hunk 2 above)
Comment (thealphadollar) [reply Dockerfile]: https://stackoverflow.com/a/729795/8184394

**Review by thealphadollar — APPROVED** [review]

**Pull request merged**